src/wallet/walletActions.js

```diff
@@ -0,0 +1,20 @@
+import Promise from 'bluebird';
+
+export const GET_WALLET = '@wallet/GET_WALLET';
+export const GET_WALLET_START = '@wallet/GET_WALLET_START';
+export const GET_WALLET_SUCCESS = '@wallet/GET_WALLET_SUCCESS';
+export const GET_WALLET_ERROR = '@wallet/GET_WALLET_ERROR';
+
+export const getWallet = (username) => {
```

Comment:
@nil1511 It looks cleaner for redux thunks since the function is returning another function. The other way it would be like this:

```
export const getWallet = (username) => (dispatch, getState, { steemAPI }) => {
  ...
}
```
which is not that easy to read.